database/src/main/java/database/exception/TableDoesNotExistException.java

```diff
@@ -0,0 +1,8 @@
+package database.exception;
+
+public class TableDoesNotExistException extends BadRequestException {
```

Comment:
If you merge `Javadocks` PR first, please merge `master` into this branch and add javadock to this class.